build/configs/rtl8720e/scripts/rlx8720e_img2_is.ld

```diff
@@ -241,10 +246,11 @@ SECTIONS
 		*(.ARM.exidx* .gnu.linkonce.armexidx.*)
 		__exidx_end = .;
 	} > KM4_BD_PSRAM_NS_NTZ
-
 	/*-----------------*/
+
 	.ram_image2.bss (NOLOAD):
 	{
+		_sbss = ABSOLUTE(.);

```

Comment:
Do we not require _ebss?
How about adding _sxxxxx and _exxxx for each section? That is the standard way used in TizenRT.

Reply:
We have _ebss already. Hence only added _sbss. 
I will verify for all other sections once again. Thank you